sidekiq_monitoring.gemspec

```diff
@@ -0,0 +1,33 @@
+$LOAD_PATH.push File.expand_path('../lib', __FILE__)
+
+# Maintain your gem's version:
+require 'sidekiq_monitoring/version'
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = 'sidekiq_monitoring_bitcrowd'
+  s.version     = SidekiqMonitoring::VERSION
+  s.authors     = ['Julian Dobmann']
+  s.email       = ['julian@bitcrowd.net']
+  s.homepage    = 'https://github.com/bitcrowd/sidekiq-monitoring'
```

Comment:
now it is https://github.com/bitcrowd/sidekiq_monitoring as you requested. But we could also change back if you want, since the gems name is different anyway.

Reply:
I personally prefer the underscored version `sidekiq_monitoring` but since the gem itself is now named differently I have no particular reason for either of both options.